replica/table.cc

```diff
@@ -2004,6 +2007,9 @@ void table::update_effective_replication_map(locator::effective_replication_map_
                           _schema->ks_name(), _schema->cf_name(), old_tablet_count, new_tablet_count);
             handle_tablet_split_completion(old_tablet_count, _erm->get_token_metadata().tablets().get_tablet_map(table_id));
         }
+
+        auto* tablet_sg_manager = dynamic_cast<tablet_storage_group_manager*>(_sg_manager.get());
+        tablet_sg_manager->update_effective_replication_map(_erm);
```

Comment:
Let's make it a virtual function instead of this dynamic_cast.

Reply:
okay